konvoy: Update 1.8 docs (#3599)
mesosphere/dcos-docs-site: pages/dkp/konvoy/1.8/upgrade/upgrade-cli/index.md

After you have the available Konvoy versions, you can upgrade your CLI by running the following command:

```bash
konvoy image upgrade --version=v1.8.0
Wrote Konvoy CLI version 'v1.8.0' to '.konvoy/cli_version'
```

After the upgrade command completes, you can start using the new Konvoy version.

### Upgrading Konvoy from v1.7.x to v1.8.0

**You must modify your `cluster.yaml` with these changes when upgrading from a previous Konvoy version:**

It is recommended to upgrade to the newest supported version of Kubernetes. Set `spec.kubernetes.version: 1.20.6` for of kind: ClusterConfiguration.

> The version of Kubernetes Base Addons changed if you use KBA, so you need to change your `configVersion` for your `configRepository`: `https://github.com/mesosphere/kubernetes-base-addons` to be `spec.addons.configVersion: stable-1.20-4.0.0` for of kind: ClusterConfiguration.

Reviewer: ```suggestion
- If you are using KBA's the version has changed. You must set your `configRepository` to `https://github.com/mesosphere/kubernetes-base-addons` and `configVersion` to `spec.addons.configVersion: stable-1.20-4.0.0` for `kind: ClusterConfiguration`.
```

> If you use Kommander, you need to change the `configVersion` for your `configRepository`: `https://github.com/mesosphere/kubeaddons-kommander` to be `spec.addons.configVersion: stable-1.20-1.4.0` for of kind: ClusterConfiguration.

Reviewer: ```suggestion
- If you use Kommander, you must set the `configVersion` to `spec.addons.configVersion: stable-1.20-1.4.0` and your `configRepository` to `https://github.com/mesosphere/kubeaddons-kommander` for `kind: ClusterConfiguration`.
```

> If you have Dispatch enabled, you need to change the `configVersion` for your `configRepository`: `https://github.com/mesosphere/kubeaddons-dispatch` to be `configVersion: stable-1.20-1.4.5` for of kind: ClusterConfiguration

Reviewer: ```suggestion
- If you have Dispatch enabled, you must set the `configVersion` to `configVersion: stable-1.20-1.4.5` and your `configRepository` to `https://github.com/mesosphere/kubeaddons-dispatch` for `kind: ClusterConfiguration`.
```
